Misc/NEWS

```diff
@@ -646,6 +646,9 @@ Library
 - bpo-29851: importlib.reload() now raises ModuleNotFoundError if the
   module lacks a spec.
 
+- bpo-30605: re.compile() no longer raises a BytesWarning when compiling a
+  bytes instance.
```

Comment:
Add "Patch by Roy Williams."

"... when compiling a  bytes instance with misplaced inline modifier." Or something like.

And usually new entries are added at the begin of a section, so they are listed in the reversed chronological order.